manifests/deps/debian.pp

```diff
@@ -3,16 +3,20 @@
 # This module manages rbenv dependencies for Debian $::osfamily.
 #
 class rbenv::deps::debian {
+  $packages = $::lsbdistcodename ? {
+    'bionic'           => [ 'libgdbm5', 'libssl1.0-dev' ],
+    default            => [ 'libgdbm3', 'libssl-dev']
+  }
   ensure_packages([
     'build-essential',
     'git',
     'libreadline-dev',
-    'libssl-dev',
     'zlib1g-dev',
     'libffi-dev',
     'libyaml-dev',
     'libncurses5-dev',
     'libgdbm-dev',
+    $packages,
```

Comment:
I'm not sure this works...inserting an array into an array? Perhaps this needs to use `merge` ?

Reply:
merge could work to, we use it now